### ISSUE TITLE
Download the model

### ISSUE THREAD

**Issue by nikitasingh06** [kovacsv/JSModeler#37]
Hey, How can I download the generated output and make changes to it?

**Comment by kovacsv** [comment]
Hi,

I recommend you to check this example:
https://rawgit.com/kovacsv/JSModeler/master/sandbox/print.html

It can export the model to STL file, so only one another step needed to make it downloadable.

**Comment by zen253230** [comment]
Hello,

I just discovered JSModeler after using Online 3d viewer and I think it's fantastic !

I used your example to export to obj an svg. Problem is I can't export the colors

Is there a way to have an mtl file generated as well?

Thank you,

**Comment by kovacsv** [comment]
No, unfortunately at the moment it can export only the geometry without material information.
